packages/packagers/css/src/CSSPackager.js

```diff
@@ -6,8 +6,24 @@ export default new Packager({
   async package({bundle}) {
     let promises = [];
     bundle.traverseAssets({
-      exit: asset => {
-        promises.push(asset.getCode());
+      exit: async asset => {
+        let parentDep;
+        bundle.traverseAncestors(asset, (node, _, {stop}) => {
```

Comment:
This packager and scope hoisting both use `traverseAncestors`, which leaks the internal nodes and internal assets from core. We should figure out from their use an appropriate public api.

Maybe for this one it's `bundle.resolveAssetDependency` or something like that.

Reply:
I think a `getIncomingDependencies` or something could work. Not sure on the naming.

Reply:
Looks like scope hoisting is also catching `assets`, not sure there is a solid abstraction with only these 2 places using it at this point.

Reply:
Based on v1, it looks like we are looking at all the parentDeps, should I not stop the traversal?

https://github.com/parcel-bundler/parcel/blob/576dba26cbc5f52d8637a4824d76d3f25fba6185/packages/core/parcel-bundler/src/packagers/CSSPackager.js#L15-L31